response_operations_ui/views/reporting_units.py

```diff
@@ -129,6 +129,12 @@ def resent_verification(ru_ref, email, party_id):
                             info='Verification email re-sent'))
 
 
+@reporting_unit_bp.route('response_chasing/<ce_id>', methods=['GET'])
+@login_required
+def response_chasing(ce_id):
+    logger.debug('Response chasing', ce_id=ce_id)
```

Comment:
Why is there an endpoint who's only job is to log out a message? I think this should be removed but I wasn't part of this card so there might be a good reason for it, though I can't think of one off the top of my head

Reply:
This is incomplete at the moment, there will be a call here to the new Reporting Service.

Reply:
@rjweeks you should lable with WIP label so that we know.